source/features/global-discussion-list-filters.tsx

```diff
@@ -5,19 +5,26 @@ import {getUsername} from '../libs/utils';
 
 function init() {
 	const defaultQuery = 'is:open archived:false ';
-	const type = location.pathname === '/issues' ? ' is:issues' : ''; // Without this, the Issues page also displays PRs
+
+	// Without this, the Issues page also displays PRs, and viceversa
+	const types: { [path: string]: string } = {
+		'/issues': 'is:issue ',
+		'/pulls': 'is:pr '
```

Comment:
I don’t think this is needed, `is:pr` is implied in `/pulls` according to GitHub. Did the previous version throw TS errors? I’d revert this change

Reply:
That's actually not what I've experienced. I've captured a couple of screenshots with the extension without these changes:

here's the default screen at [github.com/pulls](https://github.com/pulls); notice the extra `is:pr` parameter in the search field:
![Captura de pantalla de 2019-04-18 17-59-07](https://user-images.githubusercontent.com/1441704/56374635-f537a780-6203-11e9-8e67-188f569931a4.png)

and here's the screen in the `Commented` section, without the `is:pr` parameter:
![Captura de pantalla de 2019-04-18 17-59-39](https://user-images.githubusercontent.com/1441704/56374897-7abb5780-6204-11e9-9b28-2ee50b95afd4.png)

I'm still in `/pulls` but the lack of `is:pr` makes it show both Issues and PRs.

Reply:
With the commit checked out, I get the proper search results:
![Captura de pantalla de 2019-04-18 18-07-27](https://user-images.githubusercontent.com/1441704/56375062-d7b70d80-6204-11e9-9996-249e504c66ae.png)

Reply:
Indeed, that only seems to apply to `/user/repo/pulls` and not `/pulls`

e.g. no issues appear here: https://github.com/sindresorhus/refined-github/pulls?utf8=%E2%9C%93&q=